help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,160 @@
+
+{style="table-layout:auto"}
+
+## Connect to the destination
+
+[!IMPORTANT]
+To connect to the destination, you need the **View Destinations** and **Manage and Activate Dataset Destinations** [[access control permissions]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions). Read the [[access control overview]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### New Account
+
+#### Authenticate to destination
+
+To authenticate to the destination, fill in the required fields and select **Connect to destination**.
+
+To access your bucket on Experience Platform, you need to provide valid values for the following credentials:
+
+
+| **Credential** | **Description** |
+|---|---|
+|Access key|The access key ID for your bucket. You can retrieve this value from the Merkury team.|
+|Secret key|The secret key ID for your bucket. You can retrieve this value from the Merkury team.|
+|Bucket name|This is your bucket where files will be shared. You can retrieve this value from the Merkury team.|
+
+{style="table-layout:auto"}
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image4.png)
+
+#### Fill in destination details
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image6.png)
+
+-   **Name (Required)** - The name the destination will be saved under
+-   **Description** - Short explanation of the destination's purpose
+-   **Bucket Name (Required)** - Name of the Amazon S3 bucket set up on S3
+-   **Folder Path (Required)** - If subdirectories in a bucket are used a path must be defined, or '/' to reference the root path.
+-   **File Type** - Select the format Experience Platform should use for the exported files. Currently, the only file type Acxiom processing will be expecting is CSV
+
+[!NOTE]
+When selecting the CSV option, Delimiter, Quote Character, Escape Character, Empty Value, Null Value, Compression format, and Include manifest file options will be presented, the following document explains these settings in more detail configure the formatting options.
```

Comment:
there is a mention of "the following document" that explains in more detail the formatting options, but no link to the doc itself. Again, if copying material from the Acxiom destination, please make sure we do it properly and maintain useful links etc.

Reply:
Fixed

Reply:
Addressed